index.html

```diff
@@ -13,7 +13,7 @@
     <section id="homePage">
 
         <h1>Need to mix up your music?</h1>
-        <button>Let's Get Started!</button>
+        <button id="startButton" onclick="loadForm()">Let's Get Started!</button>
```

Comment:
Hullo! I haven't seen this version of click events before w adding the function directly to the html. Will the function call without writing it as an API in the js?

Reply:
Yea this is just the way I know how to do it without appending and creating the whole element in the js. We can talk to the UI team about variable names tonight